src/tests/renderer/store/game.spec.ts

```
@@ -1,11 +1,22 @@
+} from "@/renderer/store/game";
 import { RecordManager } from "@/renderer/store/record";
 import { playerURI01, playerURI02, gameSettings10m30s } from "@/tests/mock/game";
 import { createMockPlayer, createMockPlayerBuilder } from "@/tests/mock/player";
 import { GameSettings, JishogiRule } from "@/common/settings/game";
 import { PlayerBuilder } from "@/renderer/players/builder";
+import api, { API } from "@/renderer/ipc/api";
+import { Mocked } from "vitest";
+
+vi.mock("@/renderer/ipc/api");
+
+export const mockAPI = api as Mocked<API>;
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)